### PR TITLE
fix invocation tests

### PR DESCRIPTION
**Pull request by igormcoelho**

### CODE REVIEW

**Comment by jsolman** [comment]
You beat me too it. :-)

File: neo/Network/P2P/Payloads/InvocationTransaction.cs [hunk 1]
```diff
@@ -12,9 +12,7 @@ public class InvocationTransaction : Transaction
         public byte[] Script;
         public Fixed8 Gas;
 
-        public override int Size => base.Size
-            + Script.GetVarSize()           //Script
-            + Version >= 1 ? Gas.Size : 0;  //Gas
+        public override int Size => base.Size + Script.GetVarSize() + (Version >= 1 ? Gas.Size : 0);
```
Comment (jsolman) [inline neo/Network/P2P/Payloads/InvocationTransaction.cs]: better to leave it on multiple lines.

**Comment by igormcoelho** [comment]
Man, I spend 40 minutes trying to figure out why the sum was incorrect hahahah crazy precedence

File: neo/Network/P2P/Payloads/InvocationTransaction.cs (hunk 1 above)
Comment (igormcoelho) [reply neo/Network/P2P/Payloads/InvocationTransaction.cs]: my Atom editor does not consider `+ Script.GetVarSize()           //Script` as a comment... can you believe this? So I spent some time thinking line breaking was an issue too....

Comment (igormcoelho) [reply neo/Network/P2P/Payloads/InvocationTransaction.cs]: Fixed!

**Comment by shargon** [comment]
Good work!

**Pull request merged**